src/server/scripts/Spells/spell_shaman.cpp

```diff
@@ -952,6 +953,53 @@ class spell_sha_mana_tide_totem : public SpellScriptLoader
         }
 };
 
+// -30881 - Nature's Guardian
+class spell_sha_nature_guardian : public SpellScriptLoader
+{
+public:
+    spell_sha_nature_guardian() : SpellScriptLoader("spell_sha_nature_guardian") { }
+
+    class spell_sha_nature_guardian_AuraScript : public AuraScript
+    {
+        PrepareAuraScript(spell_sha_nature_guardian_AuraScript);
+
+        bool Validate(SpellInfo const* /*spellInfo*/) override
+        {
+            if (!sSpellMgr->GetSpellInfo(SPELL_SHAMAN_NATURE_GUARDIAN))
+                return false;
+            return true;
+        }
+
+        void HandleProc(AuraEffect const* aurEff, ProcEventInfo& eventInfo)
+        {
+            PreventDefaultAction();
+
+            if (Unit* target = eventInfo.GetActionTarget())
+                if (target->HealthBelowPctDamaged(30, eventInfo.GetDamageInfo()->GetDamage()))
```

Comment:
Tooltip shows "$s2%" which means value of EFFECT_1 (even if the spell doesn't have this effect it the dbc can still hold a value there)